src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Tensor.cs

```diff
@@ -3947,7 +3890,9 @@ public static Tensor<T> Hypot<T>(in ReadOnlyTensorSpan<T> x, in ReadOnlyTensorSp
         public static ref readonly TensorSpan<T> Hypot<T>(scoped in ReadOnlyTensorSpan<T> x, scoped in ReadOnlyTensorSpan<T> y, in TensorSpan<T> destination)
```

Comment:
Not strictly part of this PR, but it looks like we're missing overloads that take one of the input arguments as a scalar?

Reply:
Lets do this later as TensorPrimitives doesn't have this either.